app/components/autocomplete/slash_suggestion/app_command_parser/app_command_parser.ts

```diff
@@ -763,7 +767,15 @@ export class AppCommandParser {
             }
         }));
 
-        return ok;
+        if (Object.keys(errors).length === 0) {
+            return {};
+        }
+
+        let errorMessage = '';
+        Object.keys(errors).forEach((v) => {
+            errorMessage = errorMessage + errors[v] + '\n';
```

Comment:
How big can this error string get? Do we want to attempt to show a huge error on mobile?

Reply:
This can be very big in cases where there are many channel, static and user selectors, all with wrong values. We can always just show the first error.